bolt/include/bolt/Core/BinaryContext.h

```diff
@@ -1230,6 +1230,9 @@ class BinaryContext {
   ///
   /// Return the pair where the first size is for the main part, and the second
   /// size is for the cold one.
+  /// Modify BinaryBasicBlock::OutputAddressRange for each basic block in the
+  /// function in place so that BB.OutputAddressRange.second less
+  /// BB.OutputAddressRange.first gives the emitted size of BB.
```

Comment:
"... BasicBlock::getOutputSize() gives the emitted size of BB."

Please update the summary as well.